apps/core/test/services/users_test.exs

```diff
@@ -66,6 +66,19 @@ defmodule Core.Services.UsersTest do
       assert_receive {:event, %PubSub.UserUpdated{item: ^updated}}
     end
 
+    test "email change is properly detected" do
+      user = insert(:user)
+
+      {:ok, updated} = Users.update_user(%{name: "changed"}, user)
+      assert updated.email_changed == false
```

Comment:
you can technically just use `assert` and `refute` here although this obviously still works

Reply:
It's changed as it looks much cleaner that way.